### ISSUE TITLE
A11y tree is missing data

### ISSUE THREAD

**Issue by OrKoN** [puppeteer/puppeteer#14319]
See https://github.com/ChromeDevTools/chrome-devtools-mcp/issues/363#issuecomment-3403814558

- add the test case to the test suite
- fix the code

**Comment by OrKoN** [comment]
Landmarks landed but there could be other data omitted by default. Will need to look into that.
